<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPageSet"/>
    <addaction name="actionPrinter"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_N">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
    <addaction name="actionGoto"/>
    <addaction name="separator"/>
    <addaction name="actionAllpick"/>
    <addaction name="actionDate"/>
   </widget>
   <widget class="QMenu" name="menu_Z">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="actionAuto"/>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="actionStatus"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_N"/>
   <addaction name="menu_Z"/>
   <addaction name="menu"/>
   <addaction name="menu_S"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开(&amp;O)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为(&amp;A)...</string>
   </property>
  </action>
  <action name="actionPageSet">
   <property name="text">
    <string>页面设置(&amp;U)...</string>
   </property>
  </action>
  <action name="actionPrinter">
   <property name="text">
    <string>打印(&amp;P)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>查找(&amp;F)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>替换(&amp;R)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionGoto">
   <property name="text">
    <string>转到(&amp;G)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAllpick">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionDate">
   <property name="text">
    <string>时间/日期(&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>字体(&amp;F)...</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于记事本(&amp;A)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
